nilearn/input_data/base_masker.py

```diff
@@ -277,15 +278,22 @@ def fit_transform(self, X, y=None, confounds=None, **fit_params):
                               ' will be used.' % self.__class__.__name__)
                 return self.fit(**fit_params).transform(X, confounds=confounds)
 
-    def inverse_transform(self, X):
-        img = self._cache(masking.unmask, func_memory_level=1,
+    def inverse_transform(self, X, squeeze=False):
+        # Select squeeze as True if needed to return an image of squeezing with 
+        # last dimension "1" from a 4D format to 3D image format
+        if squeeze:
+            img = self._cache(masking.unmask, func_memory_level=1,
+            )(X, self.mask_img_)
+            img = nibabel.funcs.squeeze_image(img)
+        else:            
+            img = self._cache(masking.unmask, func_memory_level=1,
             )(X, self.mask_img_)
         # Be robust again memmapping that will create read-only arrays in
-        # internal structures of the header: remove the memmaped array
+        # internal structures of the header: remove the memmaped array                      
```

Comment:
Trailing spaces are not PEP8. You should add automatic check of PEP8 in your editor.